cni/pkg/install-cni/pkg/constants/constants.go

```diff
@@ -0,0 +1,37 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package constants
+
+// Command line arguments
+const (
+	CniNetDir            = "cni-net-dir"
+	MountedCniNetDir     = "mounted-cni-net-dir"
+	CniConfName          = "cni-conf-name"
+	KubeCfgFilename      = "kubecfg-file-name"
+	ChainedCniPlugin     = "chained-cni-plugin"
+	SkipCniBinaries      = "skip-cni-binaries"
```

Comment:
nit: I think using `CNI` instead of `Cni` would be good?
eg: 
`SkipCniBinaries` > `SkipCNIBinaries`
`CniNetworkConfig` > `CNINetworkConfig`

`KubeCaFile` > `KubeCAFile`